src/NServiceBus.AcceptanceTests/Timeout/When_timeout_storage_is_unavailable_temporarily.cs

```diff
@@ -65,14 +65,15 @@ public Endpoint()
                 {
                     config.GetSettings().Set("TimeToWaitBeforeTriggeringCriticalErrorForTimeoutPersisterReceiver", TimeSpan.FromSeconds(7));
                     config.EnableFeature<TimeoutManager>();
+                    config.DisableFeature<InMemoryTimeoutPersistence>(); // we register our custom timeout persister
```

Comment:
How about defining the storage type instead:

`config.UsePersistence<Custom, StorageType.Timeouts>()`

This would work as long as the code above runs after the default server